### PR TITLE
0.17.0 to main

### PR DESCRIPTION
**Pull request by abdulajet**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message

### Solves issue(s)

#477
#482 
#464 
#481 
#428 
#389 
#478

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 1.0.0 (Apr 20, 2021)
```
Comment (ggoldens) [inline CHANGELOG.md:1]: This should be 0.17.0. Maybe we need to revisit the date?

**Review by ggoldens — APPROVED** [review]
LGTM, just left a minor change.

File: CHANGELOG.md (hunk 1 above)
Comment (abdulajet) [reply CHANGELOG.md:1]: of course! my bad

**Review by enricop89 — APPROVED** [review]
LGTM

**Pull request merged**